modules/ROOT/pages/dw-operators.adoc

```diff
@@ -223,19 +230,19 @@ Note that `not` works in expressions such as `not (true)`, but `not(true)`
 
 You can use logical operators together. The following example uses:
 
-* `or not` as defined in the `orNot` expression
-* `and not` in `andNot`
+* `or not` as defined in the `orNot` expression.
+* `and not` in `andNot`.
 * `not` and `and not` in `notWithAndNot`.
 
 [source,dataweave,linenums]
 ----
 %dw 2.0
 output application/json
-var orNot = if (1 + 1 == 4 or not 1 == 2) {"answer": "foo"}
+var orNot = if (1 + 1 == 4 or not 1 == 2) {"answer": "true or (not false)"}
```

Comment:
```suggestion
var orNot = if (1 + 1 == 4 or not 1 == 2) {"answer": "false or (not false)"}
```
1 + 1 == 4 => false.
It looks a little confusing to answer what the if condition is resolving to.
Maybe `{"answer": "The if condition is met"}`?
I'm not sure about how should be a better example.

Reply:
fixed in 4872b22